scalajslib/worker/0.6/src/ScalaJSWorkerImpl.scala

```diff
@@ -15,14 +15,15 @@ import org.scalajs.jsenv._
 import org.scalajs.testadapter.TestAdapter
 
 class ScalaJSWorkerImpl extends mill.scalajslib.api.ScalaJSWorkerApi {
-    
+
   def link(sources: Array[File],
            libraries: Array[File],
            dest: File,
            main: String,
            testBridgeInit: Boolean, // ignored in 0.6
            fullOpt: Boolean,
-           moduleKind: ModuleKind) = {
+           moduleKind: ModuleKind,
+           useECMAScript2015: Boolean /* ignored in 0.6 */) = {
```

Comment:
This isn't ignored..

Reply:
So Scala.js 0.6.33 supports `useECMAScript2015` ?

Reply:
Yes, you're right! I'm using it 🙈

Reply:
Fixed ✅